### DISCUSSION TITLE
How disable popup for EXPLAIN Execution plan  ?

### DISCUSSION THREAD

**Discussion by andreysneg** [dbeaver/dbeaver#22159] [Q&A]
How disable popup for EXPLAIN Execution plan  ?

**Comment by LonwoLonwo** [comment]
hello @andreysneg 

What kind of a popup? Could you please provide a screen?

**Comment by andreysneg** [comment]
[https://bit.ly/47d6KAp](url)

Copy link to browser.

When click Explain icon when i must select checkboxes

**Reply by LonwoLonwo** [reply]
There is no particular setting for removing this popup. You can create an enhancement ticket for us in issues.

**Reply by andreysneg** [reply]
Its so hard. you can create ?
